src/stores/login.ts

```diff
@@ -0,0 +1,15 @@
+import { defineStore } from 'pinia';
+
+export const useLoginStore = defineStore('login', {
+  state: () => ({
+    user: {
+      email: 'test@example.com',
```

Comment:
Email key should has `null` value by default.

For testing purpose set user email key value before running test.

Reply:
Updated. Linked the login form with store state and added tests that check the store state after typing in values.